lib/gollum/public/gollum/javascript/editor/gollum.editor.js.erb

```diff
@@ -405,43 +398,6 @@
       return null;
     },
 
-
```

Comment:
Turns out the `loadFor` method wasn't doing anything anymore, since we're requiring all the language defintions in `app.js`, so was able to remove this block.

Reply:
👍